---
layout: page
excerpt: "About Me..."
---
Hello, I am currently working as a Postdoctoral Fellow at the Department of Biostatistics, University of Texas MD Anderson Cancer Center. I have previously obtained my Ph.D in Biostatistics from the University of Pittsburgh preceded by M.S. and B.S. in Applied Statistics from the University of Dhaka, Bangladesh. The focus of my research is in developing and applying machine learning algorithms to high-dimensonal genomics data. During my Ph.D., I worked as a Graduate Student Researcher (GSR) where I have gained extensive experience in working with collaborators from a wide range of research background. 


## Research Interest:

- Machine learning
- Bayesian Hierarchical modeling
- Variable selection in high-dimensional data
- Analysis of genomics data (genetical data, transcriptomics, proteomics, methylation data, etc.)
- Exploring new area of research in clinical trials, spatial statistics, survival analysis, etc.

## Research Projects

- **Meta-analytic framework using model-based clustering approach in multiple transcriptomic studies with covariate adjustment :** Unsupervised machine learning remains an important role identifying disease subtypes. The aim of this project is to extend the meta-analytic framework to sparse K-means to a meta Gaussian mixture model framework. This extension allows incorporation of information from clinical variables to control for extraneous source of variability as well consider the gene-gene dependence in an unified model.

- **Incorporating functional annotations in fine-mapping studies in the Sum of Single Effects model:** Leveraging the information from functional annotation have been shown to improve discovery power in genetic fine-mapping studies. The project is aimed to extend the Sum of Single Effects model (SuSiE) to develop an embedded model which can incorporate the information obtained from annotation into the model. Previous methods have been found to be computationally intensive and can only consider very small number of causal variants. Therefore, incorporation of annotation information into the SuSiE model could be specially relevant due to their computational simplicity enabling them to be applicable even when the number of causal variants is assumed to be moderately large. 

- **A sparse negative binomial classifier with covariate adjustment for RNA-seq data:** Developed a negative binomial model via generalized linear model framework with double regularization for gene and covariate sparsity to accommodate three key elements: adequate modeling of count data with overdispersion, gene selection and adjustment for covariate effects.

- **A sparse negative binomial mixture model for clustering RNA-seq count data:** Developed a negative binomial mixture model with lass or fused lasso gene regulation to cluster samples (small n) with high-dimensional gene features (large p). 


