---
layout: page
excerpt: "About Me..."
---
### Methodological Projects

- **Meta-analytic framework using model-based clustering approach in multiple transcriptomic studies with covariate adjustment :** Unsupervised machine learning remains an important role identifying disease subtypes. The aim of this project is to extend the meta-analytic framework to sparse K-means to a meta Gaussian mixture model framework. This extension allows incorporation of information from clinical variables to control for extraneous source of variability as well consider the gene-gene dependence in an unified model.

- **Incorporating functional annotations in fine-mapping studies in the Sum of Single Effects model:** Leveraging the information from functional annotation have been shown to improve discovery power in genetic fine-mapping studies. The project is aimed to extend the Sum of Single Effects model (SuSiE) to develop an embedded model which can incorporate the information obtained from annotation into the model. Previous methods have been found to be computationally intensive and can only consider very small number of causal variants. Therefore, incorporation of annotation information into the SuSiE model could be specially relevant due to their computational simplicity enabling them to be applicable even when the number of causal variants is assumed to be moderately large. 

- **A sparse negative binomial classifier with covariate adjustment for RNA-seq data:** Developed a negative binomial model via generalized linear model framework with double regularization for gene and covariate sparsity to accommodate three key elements: adequate modeling of count data with overdispersion, gene selection and adjustment for covariate effects.

- **A sparse negative binomial mixture model for clustering RNA-seq count data:** Developed a negative binomial mixture model with lass or fused lasso gene regulation to cluster samples (small n) with high-dimensional gene features (large p). 

- **MetaOmics: software suite for comprehensive transcriptomic meta-analysis:** Worked with lab members during my PhD to develop a comprehensive analytical pipeline and browser-based software suite to meta-analyze multiple transcriptomic studies fro various biological purposes, including quality control, differential expression analysis, pathway enrichment analysis,  differential co-expression network analysis, prediction, clustering and dimension reducton. 

- **Congurence analysis of Model Organisms (CAMO):** Worked with other members from the lab during my PhD to develop a statistical evaluation framework with functional characterizationfor for comparison of differential transcriptomic systems across model organisms or across species.

### Collaborative Projects

- Worked on a project aimed at investigating the interaction between prenatal exposure to environmental manganese exposure and gene variants on birth outcomes by performing genome-wide gene-environmental interaction study (GWEIS). As a part of the data analysis, I have carried out quality control, GWEIS, meta-analysis using METAL, imputation using michigan imputation server for the purpose of finemapping.

- Worked on a project aimed at determining the impact of sequential biopsies in patients with triple-negative breast cancer receiving neoadjuvant systemic therapy. Here, I helped in the data analysis by performing exploratory analysis, summarizing the characteristics of the variables as well as performing survival analysis, competing risk analysis as well restricted mean survival time analysis.   

- Applied Semi-competing risk model to progression of radiologic severity index associated with increased mortality in patients with acute leukemia who develop pneumonia after induction chemotherapy.

- Worked on a project to understand how cocaine experience may alter rapid eye movement (REM) sleep regulatory machinery. In this project, I analyzed RNA-seq data for a saline vs cocaine study on rat samples. As part of the analysis, I carried out quality control using FastQC, alignment using Hisat2, preprocessing the output to obtain count data and finally carrying out differential expression (DE) analysis to identify the expressed genes. 

- Applied pathway analysis using Ingenuity Pathway Analysis (IPA) software as well Gene ontology (GO) analysis to explore the mechanism of age-by-disease interactions using DNA methylation data. 

- Worked on collaborative projects to investigate the trait, state and neuro-progressive pathologies using RNA-seq data from four MDD cohorts (at various stages of disease and remission) and a control group of sample size 90. Here, I carried out quality control, alignment, preprocessing and DE analysis. 

- Analyzed NanoString data for collaborators for identifying candidtate genes for a Leiomyosarcoma study.

